SimpactApp/templates/Simpact HTML Files/updateddescription.html

```diff
@@ -0,0 +1,122 @@
+<!doctype html>
+<html>
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta charset="UTF-8">
+	<link rel="stylesheet" type="text/css" href="description.css">
+	<link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+
+	<title> Template Dashboard </title>
```

Comment:
Is it a page where volunteers can update their profiles or is it volunteer dashboard. In both cases I suggest updating both the title tag and the file name accordingly.

Reply:
Since you already merged it, you can rename the file in a separate PR.